org.lyx.LyX.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.lyx.LyX
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: lyx-sh.sh
+rename-desktop-file: lyx.desktop
+rename-icon: lyx
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=~/.lyx/:create
+  - --filesystem=xdg-config
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-documents
+  - --filesystem=/tmp/
+  - --env=PATH=/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "22.08"
+    versions: 23.08;22.08
+    directory: texlive
+    autodelete: false
+modules:
+  - name: texlive
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+  - name: lyx
+    buildsystem: autotools
+    config-opts:
+      - --enable-qt5
+      - --with-included-hunspell
+    post-install:
+      - sed -i 's/Exec=lyx %F/Exec=lyx-sh.sh/' /app/share/applications/lyx.desktop
```

Comment:
please use `desktop-file-edit` instead.

Reply:
also you remove the substitution `%F` that allow opening files.

Reply:
I have tested it locally and it works

Reply:
> please use desktop-file-edit instead.

this is not a suggestion....